### PR TITLE
Log the exception raised while trying to invoke the lambda function

### LINKED ISSUE #18: No info on why lambda invokation fails
When the warmer cannot invoke the lambda function (concurrency > 1) there's no way to actually see why.
Printing the exception would be a first help.

### PR DESCRIPTION
**Pull request by mvanbaak**
Fixes #18

While at it, I added some extra info to the log:
- full function name including version
- payload that is sent to the lambda invocation

### CODE REVIEW

File: lambdawarmer/__init__.py [hunk 1]
```diff
@@ -105,10 +106,16 @@ def _perform_fan_out_warm_up_calls(config, correlation_id, concurrency, lambda_c
...
-                Payload=json.dumps(dict(base_payload, __WARMER_INVOCATION__=(i + 1)))
+                Payload=invocation_payload
             )
         except Exception as e:
-            logger.info('Failed to invoke {} during warm up fan out'.format(LAMBDA_INFO['function_name']))
+            logger.error(
+                'Failed to invoke "%s" with event "%s" during warm up fan out. Error: "%s"',
+                function_name,
+                invocation_payload,
+                str(e)
```
Comment (robhowley) [inline lambdawarmer/__init__.py]: please use `'{}'.format` syntax for string formatting

**Comment by robhowley** [comment]
thanks for adding this. one small tweak and i'll get this out there.

File: lambdawarmer/__init__.py (hunk 1 above)
Comment (mvanbaak) [reply lambdawarmer/__init__.py]: Done.

**Comment by mvanbaak** [comment]
I pushed the fix to my branch, but github is not picking it. Want me to create a new pull-request?

**Review by robhowley — APPROVED** [review]

**Pull request merged**